microlith/run.sh

```diff
@@ -33,18 +33,25 @@ function log() {
 }
 
 # Expose all nested config variables to make it simple to see
+if [[ "${CMOS_HTTP_PATH_PREFIX:-}" == */ ]]; then
```

Comment:
Would we prefer to just sanitize the input and remove an existing trailing slash ourselves? Simplifies things in some ways (for the user).